InstabugSample/src/screens/SettingsScreen.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { Switch, StyleSheet, Text, View } from 'react-native';
+import Instabug, { BugReporting } from 'instabug-reactnative';
+
+import Button from '../components/Button';
+import ColorButton from '../components/ColorButton';
+import Screen from '../components/Screen';
+import Section from '../components/Section';
+
+const invocationEvents = [
+  { label: 'Shake', value: Instabug.invocationEvent.shake },
+  { label: 'Screenshot', value: Instabug.invocationEvent.screenshot },
+  { label: 'Two fingers swipe left', value: Instabug.invocationEvent.twoFingersSwipe },
+  { label: 'Floating button', value: Instabug.invocationEvent.floatingButton },
+  { label: 'None', value: Instabug.invocationEvent.none },
+];
+
+const colors = ['#1D82DC', 'crimson', 'olivedrab', 'gold'];
+
+function SettingsScreen() {
+  const [colorTheme, setColorTheme] = useState(Instabug.colorTheme.light);
+  const [currentColor, setCurrentColor] = useState(colors[0]);
```

Comment:
**suggestion:** Naming could be more consistent. The two states are doing the same thing but for different states (theme, color). We can rename one of them to the following

```suggestion
  const [theme, setTheme] = useState(Instabug.colorTheme.light);
  const [color, setColor] = useState(colors[0]);
```

Or:
```suggestion
  const [currentTheme, setCurrentTheme] = useState(Instabug.colorTheme.light);
  const [currentColor, setCurrentColor] = useState(colors[0]);
```

Reply:
Let's go with `theme` and `color`, I find them more readable